static/js/settings_bots.js

```diff
@@ -230,7 +229,7 @@ export function set_up_bot_edit_form(bot) {
         email: bot.email,
         full_name: bot.full_name,
         user_role_values: settings_config.user_role_values,
-        disable_role_dropdown: bot.is_owner && !page_params.is_owner,
+        disable_role_dropdown: !page_params.is_admin || (bot.is_owner && !page_params.is_owner),
```

Comment:
Do we want to allow non-admins to change the role of bots owned by them? One way can be we allow them to only change it to a role lower than their role in the hierarchy, but I think that would be unnecessary complexity and currently role of bots do not matter much as in most of the cases we allow bots tasks that are allowed to their owners.

Reply:
I think it's fine to only let admins change the role, as you've implemented.

Reply:
Yeah, that sounds correct to me.